doc/tutorials/using_windows_as_uos.rst

```diff
@@ -144,7 +72,6 @@ Create a raw disk
 
 Run these commands on the Service VM::
 
-   # swupd bundle-add kvm-host
    # mkdir /root/img
```

Comment:
I would actually much prefer we do **not** do this as root. Can this be done by a normal user? The `acrn` user is created as part of the Getting Started Guide and I'd recommend we use it instead

Reply:
change to home.